src/ucp/core/ucp_ep.c

```diff
@@ -191,6 +192,13 @@ static ucp_ep_h ucp_ep_allocate(ucp_worker_h worker, const char *peer_name)
         goto err_free_ep;
     }
 
+    ep->ext->uct_eps = ucs_malloc(UCP_MAX_SLOW_PATH_LANES * sizeof(uct_ep_h),
```

Comment:
need to allocate (and reallocate) according to key.num_lanes to reduce mem footprint